tests/rt.sh

```diff
@@ -265,15 +265,15 @@ elif [[ $MACHINE_ID = hercules ]]; then
   export PATH=/apps/spack-managed/gcc-11.3.1/miniconda3-4.10.3-un3f2xdus7rbrzgso5ketsq4gp2iociv/bin:$PATH
   export PYTHONPATH=/apps/spack-managed/gcc-11.3.1/miniconda3-4.10.3-un3f2xdus7rbrzgso5ketsq4gp2iociv/lib/python3.9/site-packages
 
-  module use /work/noaa/epic-ps/role-epic-ps/spack-stack/modulefiles
+  module use /work/noaa/epic/role-epic/spack-stack/modulefiles
   module load ecflow/5.8.4-hercules
```

Comment:
this should be changed to 

`module load ecflow/5.8.4`

Reply:
done